boxsdk/object/item.py

```diff
@@ -11,6 +12,13 @@
 from ..pagination.marker_based_object_collection import MarkerBasedObjectCollection
 
 
+class ClassificationType(TextEnum):
```

Comment:
Does this include all possible values?  It might be good to look at the metadata template and ensure that the useful enum values of that field are all represented here